tests/functional/test_install_config.py

```diff
@@ -130,6 +130,7 @@ def test_command_line_appends_correctly(script, data):
     assert "Skipping link %s" % data.find_links in result.stdout
 
 
+@pytest.mark.xfailed(reason='cannot install build requirements with an invalid index')
```

Comment:
Should have been `@pytest.mark.xfail`, sorry!

Reply:
That's fine.